### ISSUE TITLE
A problem with build single file

### ISSUE THREAD

**Issue by Sorbing** [laravel-zero/laravel-zero#268]
Hi,
Now I got an error when run building app:
```
 Build version? [v0.2.1]:
 > 0.2.2
   1. Moving application to production mode: ✔
PHP Parse error:  syntax error, unexpected 'fn' (T_FN), expecting identifier (T_STRING) in phar:///project/vendor/laravel-zero/framework/bin/box/vendor/nikic/iter/src/iter.fn.php on line 3
   2. Compile into a single file: ✔
   ErrorException  : rename(/project/utility.phar,/project/utility/builds/utility): No such file or directory
```
Last time I updated the PHP 7.2 > 7.4.
Please, help me build the project.

**Comment by owenvoke** [comment]
Hi @Sorbing, it looks like this might be an issue with the version of Box. Would you be able to try using the version of Box from laravel-zero/framework#380?

To do this, you should be able to run the following to get that version into your `vendor` directory:
```bash
wget -O vendor/laravel-zero/framework/bin/box https://github.com/laravel-zero/framework/raw/d897a7ce1e3a7733150f2d559edf154f5de7a8ee/bin/box
```

**Comment by owenvoke** [comment]
Could you also let us know what version of Laravel Zero you are using? :thinking:

**Comment by Sorbing** [comment]
Hi, @owenvoke, I no longer tried to reanimate an old project based on `laravel-zero 5.8`.
I installed `laravel-zero 6.0` and I moved all own code to new version zero. All works fine.
I think I had a special case due to the update of the PHP version. I will try to find time to restore the old project and find the source of the problem. But I can’t promise that this will happen in the near future. Thanks for your help. Have a nice day)
